packages/netdog/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
Please wrap the summary line of your commit messages around 50 characters. The ellipsis in the GitHub web view is an indication that it's too long.

I tend to follow [this style guide](https://cbea.ms/git-commit/) with the addition of a "subsystem: ..." prefix to the summary.